### ISSUE TITLE
Please put the "stars" ☆ behind the name of the route. 

### ISSUE THREAD

**Issue by TheBraveKing** [theCrag/website#2752]
Hi,
I feel it will be easier for fast orientation in route list if every line starts with the route name. And stars can be somewhere else. Or there can be a static offset (place for the the stars).

It is common to search with eyes for first letters and if they have different offset on every line it makes it difficult. Especially if the name is in different language and for you it's just a goulash of letters.

The Brave King

**Comment by brendanheywood** [comment]
Yeah I'm a fan of this, it's also why I recently made all the grades box a wider minimum width so they all line up (except for the really wide grades). Thinking it would be best to move them to the left somehow and mak them right aligned.
